modules/apps/object/object-api/src/main/java/com/liferay/object/exception/ObjectLayoutColumnFielNameException.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.object.exception;
+
+import com.liferay.portal.kernel.exception.PortalException;
+
+/**
+ * @author Sergio Alonso
+ */
+public class ObjectLayoutColumnFielNameException extends PortalException {
```

Comment:
Could you fix the typo here? "d" is missing on Field

Reply:
No need to fix it. This class won't be necessary since we're going to use NoSuchObjectFieldException